libs/hooks/open-telemetry/jest.config.ts

```diff
@@ -10,5 +10,6 @@ export default {
       },
     ],
   },
+  testEnvironment: 'node',
```

Comment:
What does this do?

Reply:
This runs Jest in a node environment, which is needed for this to work or it doesn't include some APIs that the OTel libs assume exist.

Reply:
Minor - seems default is also node 🤔  - https://jestjs.io/docs/configuration#testenvironment-string

Reply:
You're right. Odd. I though this resolved an issue, but it seems to be doing nothing now... maybe I hadn't installed something.

Reply:
removed.